LinuxWebJobHosting/appsettings.Development.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Logging": {
+  },
+  "ApplicationInsights": {
+    "InstrumentationKey": "b43e0316-395a-4c3e-aa8b-1ecd2c28c6f4",
+    "EstimatorSettings": {
+    }
+  },
+  "ConnectionStrings": {
+    "NLog.Database": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=Logs;Integrated Security=True;Persist Security Info=False;Pooling=True;MultipleActiveResultSets=True;Connect Timeout=60;Encrypt=False;TrustServerCertificate=True",
+    "Workers_Database": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=WorkerState;Integrated Security=True;Persist Security Info=False;Pooling=True;MultipleActiveResultSets=True;Connect Timeout=60;Encrypt=False;TrustServerCertificate=True"
```

Comment:
this is a problem as devs have Convention with '.'

we shall try to make this compatible... I've an Idea.
Check the ArkLegacyEnvironmentVariable something used in AddWorkerHostInfrastructure extension. we may consider replacing _ (single, not double!) with '.' there if child of ConnectionStrings or something.